### PR TITLE
Upgrade Python version to 3.8.5

### PR DESCRIPTION
**Pull request by ulgens**

### CODE REVIEW

**Comment by ulgens** [comment]
@feelepxyz hey again :wave: :slightly_smiling_face:

File: python/spec/dependabot/python/update_checker/pipenv_version_resolver_spec.rb [hunk 1]
```diff
@@ -255,8 +255,8 @@
                 to start_with("Dependabot detected the following Python")
               expect(error.message).to include("3.4.*")
               expect(error.message).
-                to include("supported in Dependabot: 3.8.4, 3.8.3, 3.8.2, "\
-                           "3.8.1, 3.8.0")
+                to include("supported in Dependabot: 3.8.5, 3.8.4, 3.8.3, "\
+                           "3.8.2, 3.8.1, 3.8.0")
```
Comment (feelepxyz) [inline python/spec/dependabot/python/update_checker/pipenv_version_resolver_spec.rb:259]: Only really need to check the first few 
```suggestion
                to include("supported in Dependabot: 3.8.5, 3.8.4, 3.8.3")
```

Comment (ulgens) [reply python/spec/dependabot/python/update_checker/pipenv_version_resolver_spec.rb:259]: Isn't it a bit misinforming?

Comment (feelepxyz) [reply python/spec/dependabot/python/update_checker/pipenv_version_resolver_spec.rb:259]: Yeah find to keep as is but we're already only testing the start of the list

**Comment by ulgens** [comment]
@feelepxyz Any ETA for this PR going live? :slightly_smiling_face:

**Comment by feelepxyz** [comment]
@ulgens should be live now, let me know if it's working. We had an issue where the wrong version was installed but this was fixed a couple of hours ago.

**Comment by ulgens** [comment]
@feelepxyz I'm still getting Python version errors.

image

**Comment by feelepxyz** [comment]
@ulgens can you send the update job id that's in the logs link?

**Comment by ulgens** [comment]
@feelepxyz `update-logs/41847919`

**Comment by feelepxyz** [comment]
Hm yes looks like this new version hasn't rolled out everywhere, looking into it.

**Comment by feelepxyz** [comment]
@ulgens could you try triggering a new update now?

**Comment by ulgens** [comment]
@feelepxyz It's okay now, thanks a lot.

**Pull request merged**